_posts/2021/2021-12-17-how-the-graphic-design-innovation-lab-boosted-my-creativity.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: How the “Graphic Design Innovation Lab” boosted my creativity
+date: 2021-12-17
+header_image: public/creativity-child.jpg
+header_overlay: true
+header_position: center
+category: events
+tags: ["design", "creativity", "workshop"]
+authors: ["Svenja"]
+about_authors: ["svoelz"]
+---
+
+On the 2nd and 3rd of December, I participated in the [Tomorrow Academy's](https://tomorrowacademy.org/){:target="_blank"} "Graphic Design Innovation Lab" workshop.
+In this post, I would like to share and reflect on some of my impressions and thoughts about the workshop.
+
+## The speakers and the workshop
+
+The twins Thomas and Martin Poschauko are multidisciplinary universal creatives who combine liberal arts, design, creative research, and academic teaching.
+As designers, they work for clients such as Sony Music, Kieler Woche, Deutsche Post, the band Labrassbanda, the Federal Ministry of Finance, or the product design label Auerberg.
+They take care of the graphic design, corporate identity, illustration, and music design.
+The authors of the multiple awards winning book ["Nea Machina - die Kreativmaschine"](https://www.poschauko.de/neamachina/){:target="_blank"} give seminars, hold lectures about "creativity", and explore the creative process in their experiments.
+As creative consultants, they already worked with companies such as MetaDesign, Red Bull, Ferrero, Maloja, Georg Thieme Verlag, Strichpunkt Design, and ServusTV.
+For more information and insights into their work, please visit their [website](https://www.poschauko.de/){:target="_blank"}.
```

Comment:
I think this section is well written but pretty advertising. I was wondering if we could shorten it a bit:

```suggestion
The twins [Thomas and Martin Poschauko](https://www.poschauko.de/){:target="_blank"} are multidisciplinary universal creatives who combine liberal arts, design, creative research, and academic teaching.
As designers, they take care of the graphic design, corporate identity, illustration, and music design for clients such as Sony Music, Kieler Woche, Deutsche Post, the band Labrassbanda, the Federal Ministry of Finance, or the product design label Auerberg.
The authors of the multiple awards winning book ["Nea Machina - die Kreativmaschine"](https://www.poschauko.de/neamachina/){:target="_blank"} give seminars, hold lectures about "creativity", and explore the creative process in their experiments.
```